### PR TITLE
chore: bump cypress to v 11

### PR DESCRIPTION
**Pull request by eschutho**
### SUMMARY

Per the cypress docs, this experimental flag should help with some of the chromium memory problems we've been seeing, but it needs at least v12 in order to do anything: 
https://docs.cypress.io/guides/references/error-messages#The-Chromium-Renderer-process-just-crashed

Bumping this version to 11. We'll do 12 in a separate PR
### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset-frontend/cypress-base/cypress.config.ts [hunk 1]
```diff
@@ -26,9 +26,9 @@ export default eyesPlugin(
     defaultCommandTimeout: 8000,
     numTestsKeptInMemory: 0,
     experimentalFetchPolyfill: true,
+    experimentalMemoryManagement: true,
     requestTimeout: 10000,
     video: false,
-    videoUploadOnPasses: false,
```
Comment (villebro) [inline superset-frontend/cypress-base/cypress.config.ts:31]: Was this removal intentional? I don't see it mentioned in the docs or the description..

**Review by villebro — COMMENTED** [review]
Nice, let's hope this fixes that flaky test! One comment on the removed config.

File: superset-frontend/cypress-base/cypress.config.ts (hunk 1 above)
Comment (eschutho) [reply superset-frontend/cypress-base/cypress.config.ts:31]: Yeah, it was flagged by typescript that it no longer exists.

**Comment by eschutho** [comment]
I checked to see if we had tried this flag before, and sure enough @rusackas had a PR that enabled it, and then removed it because it wasn't supported in v10. (they added support in v12). So I'm trying my luck to see how easily I can bump to 12. 🤞

File: superset-frontend/cypress-base/cypress.config.ts (hunk 1 above)
Comment (villebro) [reply superset-frontend/cypress-base/cypress.config.ts:31]: Oh ok, that explains why it wasn't mentioned in the docs 😄

File: superset-frontend/cypress-base/package.json [hunk 2]
```diff
@@ -29,7 +29,7 @@
   },
   "devDependencies": {
     "@types/querystringify": "^2.0.0",
-    "cypress": "^10.11.0",
+    "cypress": "^11.2.0",
```
Comment (sadpandajoe) [inline superset-frontend/cypress-base/package.json:32]: This is a bump to cypress 11 and not 12, right?

Comment (eschutho) [reply superset-frontend/cypress-base/package.json:32]: Yeah, I was running tests on 11 and then was going to to 12, but I think I'll just stick to 11 in this PR.

**Review by villebro — APPROVED** [review]
LGTM, thanks for working on this ❤️

File: superset-frontend/cypress-base/package.json (hunk 2 above)
Comment (sadpandajoe) [reply superset-frontend/cypress-base/package.json:32]: Ok this looks good then

**Review by sadpandajoe — APPROVED** [review]

**Review by geido — APPROVED** [review]

**Review by rusackas — APPROVED** [review]
I think there's a way to upgrade with a GUI using `npx` if you're not already aware. I've been curious how well that works, maybe it'll be helpful for the next bumps.

**Comment by eschutho** [comment]
> I think there's a way to upgrade with a GUI using `npx` if you're not already aware. I've been curious how well that works, maybe it'll be helpful for the next bumps.

Oh nice. I'll try that. There's also a PR already where someone did the difficult work of bumping to v13. I was thinking of reviving that PR.
